src/ui/dice.test.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from 'react';
+import { Dice } from './dice';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+test('basic', () => {
+  {
+    const dice = Enzyme.shallow(<Dice />);
+    expect(dice.html()).toContain('div');
+  }
+  {
+    const dice = Enzyme.shallow(<Dice className="custom" />);
+    expect(dice.html()).toContain('custom');
```

Comment:
`div class="custom"`
you should catch any code mutation in tests for safe refactoring of tested code

Reply:
Not sure I understand.

Reply:
expect(dice.html()).toContain('custom');
<custom class="div"> will be passed

Reply:
Correct, that's meant to pass? I'm missing something

Reply:
Okay. I see. So we should use something like hasClass?

Reply:
yes)